corehq/apps/app_manager/models.py

```diff
@@ -3904,13 +3925,6 @@ class ApplicationBase(VersionedDoc, SnapshotMixin,
     """
     Abstract base class for Application and RemoteApp.
     Contains methods for generating the various files and zipping them into CommCare.jar
-
-    There are several flavors of Applications:
```

Comment:
maybe a note here saying "see top of file for more explanation". i know i don't always check the top of the file, especially when searching by tags